site2/website/blog/2021-09-23-Apache-Pulsar-2-8-1.md

```diff
@@ -0,0 +1,148 @@
+---
+author: Hang Chen, Anonymitaet
+title: Apache Pulsar 2.8.1
+---
+
+# What’s New in Apache Pulsar 2.8.1
+
+The Apache Pulsar community releases version 2.8.1! 49 contributors provided improvements and bug fixes that delivered 213 commits.
+
+Highlights of this release are as below:
+
+- Key-shared subscriptions no longer stop dispatching to consumers when repeatedly opening and closing consumers. [PR-10920](https://github.com/apache/pulsar/pull/10920)
+
+- System topic no longer has potential data loss when not configured for compaction. [PR-11003](https://github.com/apache/pulsar/pull/11003)
+
+- Consumers are not allowed to read data that does not belong to the connected topics. [PR-11912](https://github.com/apache/pulsar/pull/11912)
+
+This blog walks through the most noteworthy changes grouped by component. For the complete list including all features, enhancements, and bug fixes, check out the [Pulsar 2.8.1 Release Notes](https://pulsar.apache.org/release-notes/#281-mdash-2021-09-10-a-id281a).
+
+# Notable bug fixes and enhancements
+
+## Broker
+
+### Precise publish rate limit takes effect as expected. [PR-11446](https://github.com/apache/pulsar/pull/11446)
+
+**Issue**: Previously, when setting precise publish rate limits on topics, it did not work.
+
+**Resolution**: Implemented a new `RateLimiter` using the `LeakingBucke` and `FixedWindow` algorithms.
```

Comment:
`LeakingBucke`, is this a typo. ?

Reply:
Good catch! I've updated it to `LeakingBucket`. Thanks!

Reply:
@eolivelli do you have more suggestions on this PR?

Reply:
@eolivelli if no more comments, I'll merge this PR.